### PR TITLE
[PLAT-296] Remove scheme trimming from step-danger

### PR DESCRIPTION
**Pull request by adborbas**
Should only be merged when https://github.com/danger/danger/pull/1253 this is merged and danger is released.

### CODE REVIEW

**Comment by orta** [comment]
that's shipped on 8.0.5

File: main.go [hunk 1]
```diff
@@ -157,3 +158,37 @@ func main() {
 	fmt.Println()
 	log.Donef("Done")
 }
+
+// trimmScheme trimms the URL if danger version is <8.0.5
+func trimmScheme(url string) string {
```
Comment (lpusok) [inline main.go]: Typo in `trimm`, should be trim.

File: main.go [hunk 2]
```diff
@@ -157,3 +158,37 @@ func main() {
...
+	rawDangerVersion, err := exec.Command("danger", "--version").CombinedOutput()
+	if err != nil {
+		log.Errorf("Could not determin danger vesion: %s", err)
```
Comment (lpusok) [inline main.go]: Typo in `determin`, should be determine.

File: main.go [hunk 3]
```diff
@@ -157,3 +158,37 @@ func main() {
 	fmt.Println()
 	log.Donef("Done")
 }
+
+// trimmScheme trimms the URL if danger version is <8.0.5
+func trimmScheme(url string) string {
+	rawDangerVersion, err := exec.Command("danger", "--version").CombinedOutput()
```
Comment (lpusok) [inline main.go]: Please use log the command executed (recommended to use the command package).

**Review by lpusok — CHANGES_REQUESTED** [review]

File: main.go (hunk 2 above)
Comment (lpusok) [reply main.go]: It is not updated.

File: main.go [hunk 4]
```diff
@@ -157,3 +157,39 @@ func main() {
 	fmt.Println()
 	log.Donef("Done")
 }
+
+// trimScheme trimms the URL if danger version is <8.0.5
```
Comment (lpusok) [inline main.go]: There is a trimm remaining :)

**Review by lpusok — CHANGES_REQUESTED** [review]
Some typos remained.

**Review by lpusok — APPROVED** [review]

**Pull request merged**